### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by ZeenatLawal**
In this PR, I:
- Added styling to all components using the Bookstore design in [Zeplin](https://zeplin.io/)

>[Live Demo](https://zeenatlawal.github.io/Bookstore-React-Redux/)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -12,8 +12,11 @@ function App() {
       <Navbar />
       <Switch>
         <Route exact path="/">
-          <BooksList />
-          <InputBook />
+          <div className="container">
```
Comment (mikenath223) [inline src/App.js:15]: - [ ] Kindly deploy the project online and include the live link in the readme.

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello  

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._


Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/App.js (hunk 1 above)
Comment (mikenath223) [inline src/App.js:15]: - [ ] Kindly deploy the project online and include the live link in the readme.

**Review by EricMbouwe — APPROVED** [review]
Hi @ZeenatLawal, 

# Status: Approved✔️
Excellent job on this project. Keep it up 👏
Your project is complete! There is nothing else to say other than... it's time to merge it 💪
Congratulations! 🎉

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**